src/main/java/hudson/views/AbstractGitValuesProvider.java

```diff
@@ -7,7 +7,8 @@
 import java.util.ArrayList;
 import java.util.List;
 
-public abstract class AbstractGitValuesProvider implements ScmValuesProvider {
+// TODO fold into GitValuesProvider

```

Comment:
Complexity here is no longer necessary, but not bothering to fully clean it up here.